records/2021-09-20.scalibility.notify-short-code.md

```diff
@@ -0,0 +1,50 @@
+# Add a short-code to Notify
+
+Date: 2021-09-20
+
+## Status
+
+**IN REVIEW**
+
+## Context
+
+Short Codes are the foundation of mobile marketing. They allow consumers to interact with media outlets, brands, retailers, governments and all kinds of other organizations via text messaging. Instead of sending text messages to a 10-digit phone number, cell phone users send it to a Short Code – a five- or six-digit number that often spells a word or brand name. A Short Code becomes a “Common Short Code” (CSC) when the same code is activated across multiple mobile phone networks, which greatly extends its reach to many more mobile phone customers (this is from txt.ca). 
+
+As a way to give Notify a brand, we looked into getting a short code for Notify. This would allow us to put the short code on Canada.ca, which would inturn lend credibility to the service. Users would know what number is associated the GC and be able to trust it for information.
+
+## Additional considerations
+
+1. Short codes are supposed to be tied to a single service. As can be seen from our application(and subsequent rejection) to AWS last year for short codes (https://docs.google.com/document/d/1IbBHE_hDtXmV03FJFeB5wRWUekzmEP4Ky2x0a_lDBtg/edit), each short code is meant to be single purpose.
+2. Notify is a software service that is run by other services. The unsubscribe feature is managed by AWS (If a customer responds to the short code with 'STOP', AWS puts the user on a do not send list). AWS allows us to remove a single user from the unsubscribe list once every 30 days.
+Eg: Immigration, Refugee and Citizenship Canada (IRCC) and Canadian Border Services Agency (CBSA) use Notify for different usecases. The same short-code would be used by both services. This leads to problems with subscriptions, as an end user might chose to unsubscribe from notifications sent by IRCC but inadvertently blocks notifications sent by CBSA.
+3. Short codes are very expensive. As of Sept 2021, a short code is 3000USD$ to setup and a 1000USD$ in recurring monthly costs. It would be cost prohibitive to get short codes per service that uses Notify.
+
+## Potential Solutions
+
+### Unsubscribe Option
+
+The main problem that arises with short codes is that an individual might unsubscibe themselves from all notifications from the Canadian Government, vs the individual service they were interested in unsubscribing from.  
+We could send a shortened link to the user to try for unsubscribing vs a 'STOP' message. The unsubscribe list would be managed through the individual Notify services.
```

Comment:
Let's note in the ADR that this is a problem we also have with long codes, it is not a new issue and is something we have to tackle even now. Hence this issue would not prevent us from adopting short code because we already have this issue unfortunately, if my understanding is correct.